src/daemon/XorgDisplayServer.cpp

```diff
@@ -128,6 +129,12 @@ namespace SDDM {
 
         if (daemonApp->testing()) {
             QStringList args;
+            QDir x11socketDir(QStringLiteral("/tmp/.X11-unix"));
```

Comment:
I can do that, but why? The way I understand the docs, QLatin1String only makes sense if there would be a constructor for QDir which accepts QLatin1String. But there isn't. Otherwise QLatin1String is just casted to QString, including the overhead. QStringLiteral on the other hand has the string stored in the program at compile time, which the QString may access directly.